cpp/include/cudf/null_mask.hpp

```diff
@@ -112,9 +112,28 @@ void set_null_mask(bitmask_type* bitmask,
                    bool valid,
                    rmm::cuda_stream_view stream = cudf::get_default_stream());
 
+/**
+ * @brief Sets a vector of pre-allocated bitmask buffers to given states in the corresponding ranges
+ * in bulk
+ *
+ * Sets `[begin_bit, end_bit)` bits of each given bitmasks to corresponding valid state. This
+ * utility is particularly advantageous if setting 16 or more bitmasks sized 2^24 bits or less.
+ *
+ * @param bitmasks Pointers to bitmasks (e.g. returned by `column_viewnull_mask()`)
+ * @param begin_bits Indices of the first bits to set (inclusive)
+ * @param end_bits Indices of the last bits to set (exclusive)
+ * @param valids Booleans indicating if the corresponding bitmasks should be set to valid or null
+ * @param stream CUDA stream used for device memory operations and kernel launches
+ */
+void set_null_masks_bulk(cudf::host_span<bitmask_type*> bitmasks,
```

Comment:
Do we like the `_bulk` prefix? It's implied by the plural `set_null_masks` IMHO.

Reply:
Updated